app/core/DeeplinkManager/Handlers/handleEthereumUrl.test.ts

```diff
@@ -65,9 +75,14 @@ describe('handleEthereumUrl', () => {
     mockNavigate.mockImplementation(() => {
       // do nothing
     });
+
+    mockIsDeeplinkRedesignedConfirmationCompatible.mockReturnValue(false);
+    mockAddTransactionForDeeplink.mockResolvedValue(
+      {} as ReturnType<typeof addTransactionForDeeplink>,
+    );
   });
 
-  it('should alerts and returns on invalid URL', () => {
+  it('alerts and returns on invalid URL', () => {
```

Comment:
S: not usre about the "returns" as I don't see any expectation of returned value.
Maybe you could use "throws" instead?
```suggestion
  it('alerts and throws on invalid URL', () => {
```